docker-tool.sh

```diff
@@ -58,13 +60,18 @@ test_docker_image() {
 shift $((OPTIND -1))
 subcommand=$1; shift
 version=$1; shift
+tesseract_languages=$1; shift
 
 case "$subcommand" in
   build)
+    build_args="--build-arg TIKA_VERSION=${version}"
+    if [[ ! -z "$tesseract_languages" ]]; then
+      build_args="$build_args --build-arg TESSERACT_LANGUAGES='${tesseract_languages}'"
+    fi
     # Build slim version with minimal dependencies
     docker build -t apache/tika:${version} --build-arg TIKA_VERSION=${version} - < minimal/Dockerfile --no-cache
     # Build full version with OCR, Fonts and GDAL
-    docker build -t apache/tika:${version}-full --build-arg TIKA_VERSION=${version} - < full/Dockerfile --no-cache
+    docker build -t apache/tika:${version}-full ${build_args} - < full/Dockerfile --no-cache
```

Comment:
Not sure how you are proposing to call this from docker-tool.sh?

Calling like this:

`./docker-tool.sh build 1.25 tesseract-ocr-eng tesseract-ocr-ita`

Will only get the the first item.

Calling like this:

`./docker-tool.sh build 1.25 'tesseract-ocr-eng tesseract-ocr-ita`

Will break due to escaping on execution

Reply:
How about to run:
```bash
./docker-tool.sh build 1.25 'tesseract-ocr-eng tesseract-ocr-ita`
```

Reply:
@mhf-ir this is the same as @dameikle has suggested...